src/nvme/ioctl.h

```diff
@@ -3811,7 +3823,23 @@ static inline int nvme_verify(nvme_link_t l, struct nvme_io_args *args)
  * Return: 0 on success, the nvme command status if a response was
  * received (see &enum nvme_status_field) or a negative error otherwise.
  */
-int nvme_dsm(nvme_link_t l, struct nvme_dsm_args *args);
+static inline int nvme_dsm(nvme_link_t l, struct nvme_dsm_range *dsm, __u16 nr_ranges, __u32 attrs,
+			   __u32 nsid, __u32 *result)
```

Comment:
As I just figured out, the `nsid
 is in cdw0, so it should be right after `nvme_link_t l`. Sorry for my mistake.